_content/articles/doubting-kalama-sutta_stephen-a-evans.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Doubting the Kālāma-Sutta: Epistemology, Ethics, and the ‘Sacred’"
+authors:
+  - "Stephen A. Evans"
+external_url: "https://journal.equinoxpub.com/BSR/article/view/8856/10308"
+drive_links:
+  - "https://drive.google.com/file/d/12IPXQcjqip1auvfJnqiHC6qCvBf2jdzb/view?usp=sharing"
+course: epistemology
+tags:
+  - ethics
+  - faith
+year: 2007
+month: may
+journal: bsr
+volume: 24
+number: 1
+pages: "91--107"
+---
+
+This article presents a different take on the Kālāma Sutta. The author suggests that the sutta is specifically about faith in a teacher and transformative practice, over the more common interpretation that the sutta is an early text on Buddhist epistemology.
```

Comment:
Should definitely link to the Kalama Sutta somewhere in this description:

`[link text](/content/canon/an3.65)`